docs/pages/bare/overview.mdx

```diff
@@ -25,52 +26,183 @@ If you've ever written a native module for React Native, you'll be surprised how
+      debug app variants. Learn how to install and configure it in your existing React Native
+      project.
+    </>
+  }
+  Icon={BookOpen02Icon}
+  href="/bare/install-dev-builds-in-bare/"
+/>
+
 <BoxLink
   title="Write native modules"
   description="Use the Expo Modules API to write native modules using Swift and Kotlin."
   Icon={Lightbulb01Icon}
   href="/modules/overview/"
 />
-<BoxLink
-  title="Use Expo CLI"
-  Icon={BookOpen02Icon}
-  description="Learn about the many benefits Expo CLI provides over @react-native-community/cli."
-  href="/bare/using-expo-cli/"
-/>
+
 <BoxLink
   title="Native project upgrade helper"
   Icon={ChevronUpDoubleIcon}
   description="View file-by-file diffs of all the changes you need to make to your native projects to upgrade them to the next Expo SDK and React Native version."
   href="/bare/upgrade/"
 />
+
+### New workflows
+
+Once your app has `expo` package installed, you can submit your app to app stores with a single command or update configure `expo-updates` library to manage remote updates to you app's code:
+
+<BoxLink
+  title="App distribution"
+  Icon={BuildArtifactIcon}
+  description="Build and submit your app to app stores with a single command."
+  href="/distribution/introduction"
+/>
+
+<BoxLink
+  title="Install expo-updates"
+  Icon={BuildArtifactIcon}
+  description={
+    <>
+      Learn how to install and configure <CODE>expo-updates</CODE> to manage remote updates and
+      enable PR previews.
+    </>
+  }
+  href="/bare/installing-updates/"
+/>
+
+### New mindset
+
+The following helps with your project's long term maintainability, native code maintenance, and easier upgrades:
+
 <BoxLink
   title="Adopt Prebuild"
   Icon={RefreshCw02Icon}
   description="Learn how to simplify maintaining your native projects by generating them on demand from configuration."
-  href="/guides/adopting-prebuild"
+  href="/guides/adopting-prebuild/"
 />
+
 <BoxLink
-  title="App distribution"
-  Icon={BuildArtifactIcon}
-  description="Build and submit your app to the app store with a single command."
-  href="/distribution/introduction"
+  title="Expo Router"
+  Icon={RouterLogo}
+  description="Expo Router is a file-based routing library that offers advantages such as organized navigation hierarchy, automatic deep linking support, and more."
+  href="/router/introduction/"
 />
+
+## Common questions
+
+<Collapsible summary="How long will it take to adopt Expo in my existing React Native project?">
+
+Adopting Expo doesn't have to be done in one step. You can start with _quick wins_ and then move to tackle more complex parts. You can also pick and choose which features you want to adopt at the beginning of this process.
+
+</Collapsible>
+
+<Collapsible summary="What will I gain from using Expo in my React Native App?">
+
+Adopting Expo tools in your existing React Native app will help you maintain long-term maintenance support, easier native upgrades, and a better developer experience.
```

Comment:
Would like something here that's brief but more specific. I think adopting wording along the lines of our front-page copy about build, test, deploy, might be good, or something about filling in the blanks with a framework. I'm working on some related verbiage and may have a more concrete suggestion soon.

Reply:
though I guess frameworks are covered below

Reply:
Yes, the framework is covered in `Why did React Native recommend using Expo?`.
I'll wait for your suggestion.

Reply:
Maybe... `Adopting Expo tools into your existing React Native app can help you develop faster with the Expo SDK [link], streamline native code maintenance and upgrades [link to CNG], deploy faster with EAS Update [link], and more.`